### ISSUE TITLE
Warning: DNS doesn't seem to work properly -  daemon/session/dns/RecursionCheck

### ISSUE THREAD

**Issue by oamm** [telepresenceio/telepresence#3467]
**Description:**
When running the telepresence connect command, an immediate warning is encountered, and subsequent attempts to intercept services within the cluster fail. The warning indicates a DNS issue, preventing proper functionality.

**Steps to Reproduce:**

Execute the command: telepresence connect
Observe the warning message and subsequent service interception failure.
Error Details:

``` sh
telepresence connect
Launching Telepresence User Daemon
Launching Telepresence Root Daemon

Connected to context kubernetes-admin@kubernetes, namespace lynx (https://192.168.100.9:6443)
Warning: DNS doesn't seem to work properly
```
**Environment Details:**

- Telepresence Version:
-- Client: v2.18.0
-- Root Daemon: v2.18.0
-- User Daemon: v2.18.0
-- Traffic Manager: not connected
- Windows Version:
-- Caption: Microsoft Windows 10 Pro
-- Version: 10.0.19045
-- ServicePackMajorVersion: 0
-- OSArchitecture: 64 bits
-- WindowsDirectory: C:\Windows
- Server Version:
-- Debian GNU/Linux 12 (bookworm)
-- Kernel Version: 6.1.0-16-amd64
- Kubernetes Version:
-- Client Version: v1.28.5
-- Kustomize Version: v5.0.4-0.20230601165947-6ce0bf390ce3
-- Server Version: v1.28.5

**Expected Behavior:**
Telepresence should connect successfully without any DNS warnings, and services within the cluster should be interceptable.

**Actual Behavior:**
Immediate DNS warning is encountered, and services cannot be intercepted within the cluster.

**Additional Information:**
Any relevant logs, error messages, or configurations that might assist in identifying the issue.
[telepresence_logs.zip](https://github.com/telepresenceio/telepresence/files/13775597/telepresence_logs.zip)

**Comment by thallgren** [comment]
I believe that the real problem might be this:
```
 daemon/session/network : failed to add subnet 10.96.0.0/12: failed to create route 10.96.0.0/12 via 10.96.0.0 dev tel0, gw 10.96.0.1: Correcto
 ```
 which makes me wonder if the underlying tun-device expects an english response on some low-level call here.

Can you please set the log-level to debug, try again, and the convey the log output? You set this in `%APPDATA%\telepresence\config.yml` like this:
```yaml
logLevels:
  userDaemon: debug
  rootDaemon: debug
```

**Comment by oamm** [comment]
Hi, there is the logs after set telepresence in debug mode, 
[telepresence_logs.zip](https://github.com/telepresenceio/telepresence/files/13781432/telepresence_logs.zip)

**Comment by thallgren** [comment]
Closed by #3489
